spec/datetimeformat.html

```diff
@@ -500,7 +504,10 @@
           The array that is the value of the "nu" property of any locale property of [[LocaleData]] must not include the values "native", "traditio", or "finance".
         </li>
         <li>
-          [[LocaleData]][locale] must have hourCycle property with String values for all locale values.
+          [[LocaleData]][locale].hc must be the array [*null*, *"h11"*, *"h12"*, *"h23"*, or *"h24"*] for all locale values.
```

Comment:
It's required to be an array per https://tc39.github.io/ecma402/#sec-internal-slots:
> The value of each of these properties must be a non-empty array of those values defined in Unicode Technical Standard 35 for the given key that are supported by the implementation for the given locale, with the first element providing the default value.

(I also have a patch *somewhere* to make this use Lists instead of Arrays, at least I think so...)